grafana docker/provisioning/datasources/datasources.yaml

```diff
@@ -0,0 +1,11 @@
+apiVersion: 1
+
+datasources:
+  - name: MySQL
+    type: mysql
+    url: db:3306
+    database: airbnbProjectDB
+    user: root
+    secureJsonData:
+      password: P@ssWordAirbnb2024
```

Comment:
please put your password in an .env and reference it instead of commiting it

Reply:
Might have to make a variable.